internal/cli/internal/command/services/postgres/postgres.go

```diff
@@ -0,0 +1,27 @@
+// Package postgres implements the postgres command chain.
+package postgres
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+)
+
+// New initializes and returns a new postgres Command.
+func New() (cmd *cobra.Command) {
+	const (
+		// TODO: document command
+		long = `
```

Comment:
don't we need docs here?

Reply:
There's going to be a lot of work going into this over the next two weeks and given it's hidden behind the Dev flag I'm not super concerned about the docs quite yet.